src/manipulators/no_implicit_any_manipulator.ts

```diff
@@ -141,19 +146,49 @@ export class NoImplicitAnyManipulator extends Manipulator {
   ): Set<AcceptedDeclaration> {
     const declarations = new Set<AcceptedDeclaration>();
 
-    nodeDiagnostics.forEach(({node: node}) => {
-      if (Node.isIdentifier(node)) {
-        node
-          .getSymbol()
-          ?.getDeclarations()
-          ?.filter(
-            declaration =>
-              Node.isVariableDeclaration(declaration) ||
-              Node.isParameterDeclaration(declaration)
-          )
-          ?.forEach(declaration => {
-            declarations.add(declaration as AcceptedDeclaration);
-          });
+    nodeDiagnostics.forEach(({node: node, diagnostic: diagnostic}) => {
+      switch (diagnostic.getCode()) {
+        case ErrorCodes.ReturnTypeImplicitlyAny: {
+          // TODO: Move console log functionality to a logger class.
+          console.log(
+            chalk.cyan(`${node.getSourceFile().getFilePath()}`) +
```

Comment:
extract to a function?

logError(node, customMsg)?

Reply:
Yes, definitely! I'm not addressing this in this particular PR because I've addressed it in #43 , which sets up the Logger class to take care of this. Once one PR is merged into master, I'll rebase and update the other branch to ensure that in the end, the logger's method is the one being used here.